message/pb/message.proto

```diff
@@ -38,9 +38,28 @@ message Message {
     BlockPresenceType type = 2;
   }
 
+  message BlockManifest {
+    message BlockManifestEntry {
+      message ChunkedBlockManifestEntry {
+        bytes cid = 1 [(gogoproto.customtype) = "Cid", (gogoproto.nullable) = false];
+        int32 blockStartIndex = 2;
+        int32 blockEndIndex = 3;
+      }
+      bytes proof = 1;
+      int64 fullBlockStartIndex = 2;
+      int64 fullBlockEndIndex = 3;
```

Comment:
inclusive or exclusive? why not "block start" and "block length" (same above).

Reply:
No strong reason, it's not clear I even need both so I can just remove one later.

Similarly, I think what indices and numbers are used in the chunked parts will likely change